browser/src/slideshow/RenderContext.ts

```diff
@@ -131,7 +131,7 @@ class RenderContextGl extends RenderContext {
 		gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.LINEAR);
 
 		if (image instanceof HTMLImageElement)
-			console.log(`Texture loaded:`, image.src);
+			console.debug(`Texture loaded successfully`);
```

Comment:
Why has `image.src` been taken out of the debugging log here?

Reply:
It's big string (decoded image) which is not useful for most of time.

Reply:
ah ok, makes sense - is this debug output useful at all in that case? Presumably there'd be an exception of some kind in the case of failure(?)